packages/actor-init-sparql/test/HttpServiceSparqlEndpoint-test.ts

```diff
@@ -122,14 +149,27 @@ describe('HttpServiceSparqlEndpoint', () => {
         moduleRootPath,
         env,
         defaultConfigPath,
-        exit);
-
-      expect(exit).toHaveBeenCalledWith(1);
-      expect(stderr.toString()).toBe(`${HttpServiceSparqlEndpoint.HELP_MESSAGE}Server running on http://localhost:3000/sparql\n`);
+        exit).then(res => {
+        expect(exit).toHaveBeenCalledWith(1);
+        expect(stderr.toString()).toBeDefined();
```

Comment:
Where possible, I would check for an exact string in these unit tests. Because now it's still possible that the stderr value is defined, but is the wrong value.